doc/articles/wpf-migration.md

```diff
@@ -0,0 +1,60 @@
+
+Advancements such as WebAssembly (WASM), a low-level bytecode for the web, have already become a viable, performant way to deliver your modern .NET applications.
+
+## Migrating the codebase
+
+The Uno Platform not only allows you to maximize reuse of C#, XAML, and .NET investments, but also enables your team to naturally leverage their existing skillset. Alongside your existing investments, you may decide to further enhance your app, refreshing aspects to make use of new capabilities in the platform. Regardless, your goal is to consider what role your application will play several years or more into the future. Depending on your scenario, it may make sense to enhance the user interface layout for different form factors. In this article, the primary focus will be on rewriting as little as possible.
+
+> [!TIP]
+> Read the UWP/WinUI & Uno documentation first to better understand the capabilities of the platform. 
+
+## Limitations
+
+The vast majority of your codebase will simply "move across". Existing XAML structure, business logic, and view models can simply be reused. You'll need to recompile old code, and won't have access to framework-only APIs. You may encounter APIs that aren't implemented or controls where a subset of the functionality is presently unsupported. Additionally, the following contain substantial enough differences from WPF to require extra research upfront:
+* Navigation
+* Advanced binding functionality with x:Bind
+* Specific framework capabilities
+* Modern APIs are async only
+* Uno has a streamlined set of XAML capabilities for performance reasons
+
+## Understanding beforehand
+
+Along the journey to bring your WPF app to the Web, you should become familiar with the Uno Platform documentation. Your team should thoroughly understand what is supported (and where) before you build a feature around it. Learn about:
+* Get started
+* How-tos and tutorials
+* Development deep-dives
+* Reference
+
+## Architecture Considerations
+
+### MVVM Libraries
+Several of the popular MVVM libraries like MVVMLite, Prism, and the Windows Community Toolkit implementation have already been ported to Uno. If your existing app leverages something in-house, there's a chance it will just work. All in all, Uno has made significant strides to improve porting MVVM applications.
+
+### Navigation
+Apps leveraging Page-based navigation are relatively straightfoward to migrate. If your app has a different navigation stack, there may be additional implementation changes to consider.
+
+### Custom Controls
+One of the strengths of Uno is having access to the same, opinionated set of battle-tested controls that exist on Windows everywhere. For that reason, we reccomend determining whether there is an "inbox" equivalent to your custom controls. Depending on your app's design goals, it may be desirable to switch to UX patterns more accomodative of your users' device categories.
```

Comment:
```suggestion
One of the strengths of Uno is having access to the same, opinionated set of battle-tested controls that exist on Windows everywhere. For that reason, we recommend determining whether there is an included framework equivalent to your custom controls. Depending on your app's design goals, it may be desirable to switch to UX patterns more accomodative of your users' device categories.
```

I've never seen the term "inbox" used to mean this, I think the alternative wording would be clearer for most readers.